docs/policy.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How to opt-out of sending personal data to IntelliCode services from Visual Studio
+description: Learn how to opt-out of sending personal data to IntelliCode Services in Visual Studio.
```

Comment:
normalize capitalization of `IntelliCode services` or `IntelliCode Services` across the doc